proposals/Level-1.md

```diff
@@ -317,42 +301,41 @@ exception tags.
 This section describes change in the
 [Modules document](https://github.com/WebAssembly/design/blob/master/Modules.md).
 
-### Exception index space
+### Event index space
 
-The _exception index space_ indexes all imported and internally-defined
-exceptions, assigning monotonically-increasing indices based on the order
-defined in the import and exception sections. Thus, the index space starts at
-zero with imported exceptions followed by internally-defined exceptions in
-the [exception section](#exception-section).
+The `event index space` indexes all imported and internally-defined events,
+assigning monotonically-increasing indices based on the order defined in the
+import and event sections. Thus, the index space starts at zero with
+imported events, followed by internally-defined events in the [event
+section](#event-section).
 
-The exception index space defines the (module) static version of runtine
-exception tags. For exception indicies that are not imported/exported, the
-corresponding exception tag is guaranteed to be unique over all loaded
+The event index space defines the (module) static version of runtine
+event tags. For event indicies that are not imported/exported, the
+corresponding event tag is guaranteed to be unique over all loaded
 modules.
 
-For exception indices imported/exported, unique exception tags are created for
-each unique name imported/exported, and are aliased to the corresponding
-exception index defined in each module.
+For event indices imported/exported, unique event tags are created for each
+unique name imported/exported, and are aliased to the corresponding event index
+defined in each module.
 
 ## Changes to the binary model
 
 This section describes changes in
 the
 [binary encoding design document](https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md).
 
-
 ### Data Types
 
 #### except_ref
 
-An exception reference points to an exception value. The size
-is fixed, but unknown in WebAssembly (the host defines the size in bytes).
+An exception reference points to an exception. The size is defined by
```

Comment:
See earlier comment. I would drop any reference to size, given that it is not observable in any way.